src/app/models/Order.ts

```diff
@@ -23,6 +24,7 @@ const OrderSchema = new Schema<IOrder>(
     city: { type: String, required: true },
     cartProducts: { type: Object, required: true },
     paid: { type: Boolean, default: false },
+    canceled: { type: Boolean, default: false },
```

Comment:
@MamdouhEnahla  I added this field to mark canceled orders. I need it on admin panel